src/service/ComposeService.ts

```diff
@@ -227,6 +228,7 @@ export class ComposeService {
                                     command: brokerCommand,
                                     ports: resolvePorts([{ internalPort: 7902, openPort: n.brokerOpenPort }]),
                                     stop_signal: 'SIGINT',
+                                    stop_grace_period: '20s',
```

Comment:
Same here.  good be to be configurable.

Reply:
Now it's possible to overwrite defaults. You can also define it for other services (not only broker and server) if it will be needed via custom preset.